File: py/19.py
from datetime import date

start = date(1901, 1, 1).toordinal()
end = date(2000, 12, 31).toordinal()

sundays = 0

for i in xrange(start, end):
    date = date.fromordinal(i)
    if date.weekday() == 6 and date.day == 1:
        sundays += 1

print sundays

File: js/19.js
day = 0, sundays = 0;

while (++day) {
    date = new Date(1901, 0, day);
    if (date.getYear() > 100)
        break;
    if (date.getDate() == 1 && date.getDay() == 6)
        ++sundays;
}

console.log(sundays);
